src/material/datepicker/datepicker.md

```diff
@@ -290,6 +290,10 @@ from `@angular/core`. If you want to override it, you can provide a new value fo
 export class MyApp {}
 ```
 
+**Note:** if you're using the `MatDateFnsModule`, you have to provide the data object for your
+locale to `MAT_DATE_LOCALE`, instead of the locale code. Locale data for `date-fns` can be imported
+from `date-fns/locale`.
+
```

Comment:
There is also MAT_DATE_FORMATS to provide (Otherwise an error occurs: RangeError: Format string contains an unescaped latin alphabet character `o`). Full working example:
```ts
import { DateFnsAdapter, MAT_DATE_FNS_FORMATS } from '@angular/material-date-fns-adapter';
import { DateAdapter, MAT_DATE_FORMATS, MAT_DATE_LOCALE } from '@angular/material/core';
import { deAT } from 'date-fns/locale';

  { provide: DateAdapter, useClass: DateFnsAdapter },
  { provide: MAT_DATE_LOCALE, useValue: deAT },
  { provide: MAT_DATE_FORMATS, useValue: MAT_DATE_FNS_FORMATS },
```